docs/modules/user-guide/partials/proc_developing-with-devfiles.adoc

```diff
@@ -0,0 +1,36 @@
+[id="proc_developing-with-devfiles_{context}"]
+= Developing with devfiles
+[role="_abstract"]
+
+Develop a node.js “Hello World” application, using the devfile specification. Developing this application introduces you to how a devfile automates and simplifies your development process.
```

Comment:
```suggestion
Use the devfile specification to develop a node.js “Hello World” application. Developing this application introduces you to how a devfile automates and simplifies your development process.
```
- Reword the first sentence to make it more "devfile"-centric. My suggestion also creates a stronger transition to the next sentence by putting the 2 uses of "application" in closer proximity.